index.js

```diff
@@ -82,18 +81,24 @@ function Tallahassee(app, options = {}) {
   }
 
   function load(resp) {
-    let initialized, pending, currentPageXOffset, currentPageYOffset;
+    let pending, currentPageXOffset, currentPageYOffset;
     let elementsToScroll = () => {};
     const stickedElements = [];
 
-    compile();
-
     const headers = getHeaders(resp.request);
+    const document = Document(resp, agent.jar);
     const window = Window(resp, {
       fetch: Fetch(agent, resp),
+      get document() {
```

Comment:
What is this I don't even

Reply:
instead of `Object.defineProperty(window, "document", { get() { return document; } }`. ES6.